src/main/kotlin/org/wycliffeassociates/otter/common/domain/md/ParseMd.kt

```diff
@@ -0,0 +1,56 @@
+package org.wycliffeassociates.otter.common.domain.md
+
+import java.io.BufferedReader
+
+// TODO: Add Help type enum to HelpResource? (tn, tq)
+data class HelpResource(var title: String, var body: String)
+
+class HelpResourceList: ArrayList<HelpResource>()
```

Comment:
This should be of type `List` instead of `ArrayList`, so you're not locked into this implementation. Even if you want to build it using a mutable list such as `ArrayList`, it should be presented to consumers as `List`.

Reply:
Note that if you wish to return a list that has the ability to add elements, the List interface is immutable in kotlin, unlike Java, and you'd need to do MutableList instead.